script/openqa-bootstrap-container

```diff
@@ -79,6 +84,6 @@ systemctl start systemd-nspawn-openqa@$CONTAINER_NAME
 # ensure that the container is really running
 # ignore expected errors about 'Failed to create bus connection: Protocol error' and restarting error
 while ! timeout -s9 2 systemd-run -qPM $CONTAINER_NAME /bin/bash -c whoami /dev/null 2>&1 ; do systemctl restart systemd-nspawn-openqa@$CONTAINER_NAME.service || true ; sleep 3 ; done
-systemd-run -qPM $CONTAINER_NAME /bin/bash -c '/usr/share/openqa/script/openqa-bootstrap'
+systemd-run "$systemd_run_params" /bin/bash -c '/usr/share/openqa/script/openqa-bootstrap'
```

Comment:
Does it really work to pass these parameters as one argument? Likely you'll have to get rid of the quoting. Note that having white-spaces within the `CONTAINER_NAME` didn't work before as well. To support white-spaces you could of course just use an array (and then keep the quotes).

Reply:
it does work like this as far as I can observe. Trying just `systemd-run "-qh"` seems to do the same as `systemd-run -qh`.

Reply:
Of course `systemd-run "-qh"` is equivalent to the unquoted version. My point was whether `… "-qM some-container-name"` is equivalent to `… -qM some-container-name"`. In the first case everything is passed as one argument (including the container name) and in the second case the container name is passed as separate argument. I suspect that only the last invocation will work, especially if the first character of the container name is also a valid short option.

Reply:
right. I changed to a bash array now.